packages/kununu-footer/package.json

```diff
@@ -5,33 +5,25 @@
   "main": "dist",
   "author": "kununu",
   "license": "ISC",
-  "devDependencies": {
-    "@kununu/kununu-theme-v2": "2.0.2",
-    "babel-cli": "6.26.0",
```

Comment:
Do we not need this?

Reply:
This was moved to the root package.json dependencies, lerna lets you install it there and then run commands that use it.